packages/blockchain-wallet-v4-frontend/src/data/components/usedAddresses/reducers.js

```diff
@@ -1,4 +1,5 @@
 import { assocPath } from 'ramda'
+import { Remote } from 'blockchain-wallet-v4/src'
 import * as AT from './actionTypes'
 
 const INITIAL_STATE = {}
```

Comment:
initial state should be {addresses: Remote.NotAsked, visible: false}

Reply:
since the state tree for the component is keyed off of walletIndexes, I cant really set initial state here because i do not yet know how many wallets the user has in their account.  see screenshot.
![screen shot 2018-05-31 at 10 48 10 am](https://user-images.githubusercontent.com/6364918/40789327-2bbadb7a-64c0-11e8-9ab5-9bde7b7f6869.png)

if you have suggestions for better design, let me know.